examples/scala/src/main/scala/example/Streaming.scala

```diff
@@ -107,8 +107,36 @@ object Streaming {
     println("Delta Table after streaming upsert")
     deltaTable.toDF.show()
 
+    // Streaming append and concurrent repartition using  data change = false
+    // tbl1 is the sink and tbl2 is the source
+    println("############ Streaming appends with concurrent table repartition  ##########")
+    val tbl1 = "/tmp/delta-table4"
+    val tbl2 = "/tmp/delta-table5"
+    val numRows = 10
+    spark.range(numRows).write.mode("overwrite").format("delta").save(tbl1)
+    spark.read.format("delta").load(tbl1).show()
+    spark.range(numRows, numRows * 10).write.mode("overwrite").format("delta").save(tbl2)
+
+    // Start reading tbl2 as a stream and do a streaming write to tbl1
+    // Prior to Delta 0.5.0 this would throw StreamingQueryException: Detected a data update in the source table. This is currently not supported.
+    val stream4 = spark.readStream.format("delta").load(tbl2).writeStream.format("delta")
+      .option("checkpointLocation", new File("/tmp/checkpoint/tbl1").getCanonicalPath)
```

Comment:
nit: This file is starting to look long and messy. can you break down the individual examples as separate methods so that each of them standout clearly. Should not be a lot of work. Just pass the sparksession into each function.